pkg/universe.dagger.io/x/gabriel.gazola@poli.ufrj.br/debian/debian.cue

```diff
@@ -0,0 +1,49 @@
+// Base package for Debian Linux
+package debian
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// Build a Debian Linux container image
+#Build: {
+
+	// Debian version to install.
+	version: string | *"stable@sha256:7ec7bef742f919f7cc88f41b598ceeb6b74bcb446e9ce1d2d7c31eb26ccba624"
```

Comment:
I would prefer defaulting to slim version of image because of faster build times and smaller images. Also if you lock it to bspecific version with hash then it would be nice to use more detailed tag like `bullseye-20220418-slim` so it is easier to know which version it is (like alpine package does).

Reply:
@gabriel-milan Do you know if there is a multi-arch (at least `amd64` and `arm64` for Debian that we can use here?